### ISSUE TITLE
Add alumni section

### ISSUE THREAD

**Issue by kev-chien** [pioneers/website#493]
Somehow have some section (or different page) that shows alumni PMs/leadership (and/or members).

As to how far back to go for collecting the data, it may depend on whether the data can be easily found (old commits).

That being said this does conflict with the current way we have advisors set up (who seem to be alums who still stick around). Maybe web alums can help clarify.

**Comment by dwai213** [comment]
These definitions are pretty loose and I think the current leadership team's opinions should be considered as well, but:

Advisors: We can keep this section exclusive to faculty advisors

Alumni: As is, it is indeed a group of people that have had a history of being involved with the organization, maybe in particular towards the beginning of the organization. However, there are up and coming alumni that are as just as a big part of the organization!
Moreover, having an alumni section helps employers, if they cared enough, confirm that this student has been a part of the club as well as being an institutional way to pay homage to our heavy lifters.

The next question would most likely be who makes the cutoff to be a part of this alumni section.
Something that's fair could be PMs and above, 3 year staff and above, (2 year and above if you are a transfer), leadership members, and then the occasional exception. Another useful tool might be the [Alumni Contacts](https://docs.google.com/spreadsheets/u/1/d/1Hj6gtoP46KPEwpHYfbNqKEPYkd2iVzoFo-fqdkDHCQw/edit?usp=drive_web&ouid=104043912878571565843) spreadsheet and the alumni that continue to update their information in the spreadsheet can make it to the alumni section.

In short, I think the alumni section doesn't need to be pruned, but can grow. The webpage design would either suggest that the alumni section would need to be a little more tucked away so that we wouldn't have a giant page of alumni pictures or the alumni section can be simply a table of name, year, current job

**Comment by viviehn** [comment]
We should update alumni list~ 

(Perhaps worth creating a new issue/task)
